scripts/commands/checkLinks.ts

```diff
@@ -141,6 +147,8 @@ async function determineCurrentDocsFileBatch(
     // Ignore dev version
     "!docs/api/{qiskit,qiskit-ibm-provider,qiskit-ibm-runtime}/dev/*",
     "!public/api/{qiskit,qiskit-ibm-provider,qiskit-ibm-runtime}/dev/*",
+    // Ignore Qiskit release notes
+    "!docs/api/qiskit/release-notes/*",
   ];
   const toLoad = [
     // The 0.46 docs are used by release notes for APIs that were removed in 1.0.
```

Comment:
I think you'll temporarily need to load these files for _all_ Qiskit release notes. We can clean that up after https://github.com/Qiskit/documentation/pull/865

For now, you could make this list a file constant and pass it as an arg for Qiskit with `extraFilesToLoad`.

Reply:
Good catch! I added them to the `toLoad` directly inside the release notes' if statement. Given that is going to be super provisional, we can remove it easily that way without modifying the current version. To avoid any link breaking I had to add 1.0 and other links like 0.45 besides the 0.46 version.